__tests__/integration/one-app.spec.js

```diff
@@ -1668,3 +1705,323 @@ describe('heapdump', () => {
     expect(size).toBeLessThanOrEqual(100 * oneMegabyte);
   });
 });
+
+describe('Routes sanity checks', () => {
```

Comment:
```suggestion
describe('Routes confidence checks', () => {
```

The term "sanity check" is ableist